From e7af8c93b9dfc8914c147d1ef89a2f0482f77755 Mon Sep 17 00:00:00 2001
From: "Subhransu S. Prusty" <subhransu.sekhar.prusty@xilinx.com>
Date: Fri, 4 Sep 2020 03:55:37 -0700
Subject: [PATCH 1569/1852] ASoC: xlnx: Decouple sound card config dependency
 form all interfaces

commit f296a13205add18e97ab563895d230bf45d3efaf from
https://github.com/Xilinx/linux-xlnx.git

Allow selection of sound card config even when one of the interfaces
(i2s/hdmi/sdi) is enabled.

Signed-off-by: Subhransu S. Prusty <subhransu.sekhar.prusty@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/Kconfig | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/sound/soc/xilinx/Kconfig b/sound/soc/xilinx/Kconfig
index e3239673b0e9..c4d9a161e269 100644
--- a/sound/soc/xilinx/Kconfig
+++ b/sound/soc/xilinx/Kconfig
@@ -48,9 +48,7 @@ config SND_SOC_XILINX_SPDIF
 
 config SND_SOC_XILINX_PL_SND_CARD
 	tristate "Audio support for the the Xilinx PL sound card"
-	depends on SND_SOC_XILINX_AUDIO_FORMATTER
-	depends on SND_SOC_XILINX_I2S
-	depends on SND_SOC_XILINX_SDI
+	depends on SND_SOC_XILINX_AUDIO_FORMATTER || SND_SOC_XILINX_I2S || SND_SOC_XILINX_SDI
 	select SND_SOC_HDMI_CODEC
 	help
 	  Select this option to enable Xilinx PL sound card
-- 
2.31.1

